webpack.common.js

```diff
@@ -7,7 +7,7 @@ const webpack = require('webpack');
 /*::
 type Target = {|
     name: string, // the name of output JS/CSS
-    entry: string, // the path to the entry point
+    entry: string | Object, // the path to the entry point
```

Comment:
It's too bad there aren't any existing flow type definitions for webpack.

Reply:
@kevinbarabash There seems to be an open issue in https://github.com/flowtype/flow-typed/issues/165. Also webpack has JSON Schema([schemas/webpackOptionsSchema.json](https://github.com/webpack/webpack/blob/master/schemas/webpackOptionsSchema.json)) and maybe we can convert it using `json-schema-to-flow-type`. I wonder there is any way to import JSON schema to flow directly.

Reply:
I wouldn't worry about it for this PR.